tests/dtypes/test_text.py

```diff
@@ -466,9 +466,7 @@ def test_text_unknown_language(
     assert select_by_prop.status_code == 200
     assert len(select_by_prop.json()['_data']) == 1
     assert select_by_prop.json()['_data'] == [{
-        'name': {
-            'C': 'LT',
-        }
+        'name': 'LT'
```

Comment:
If explicit form is requested `name@C` we must always return data in explicit form to, here we should get:

```python
'name': {
    '': 'LT',
}
```